tools/rosbag/README.md

```diff
@@ -16,6 +16,11 @@ rosbag record --file-name record.yaml
 ```yaml
 # topic_name: freq(double)
 # topic_name: -1 (it will be recorded at publish freq)
+#           : 0 (it will not be recorded)
+# For topic name same regular expressions cli can be used: ex. "/realsense(.*)" : 0
+# [used by default rosbag](http://wiki.ros.org/rosbag/Commandline)
+# if there is a topic in a namespace which has been excluded using regular expressions: 
+#   to record that topic put that topic before the regular expression
```

Comment:
If the default behavior is now that topics are recorded, the description (line 11) should be updated

Reply:
if you only mention the file, it will only record the topic that is in the file. For our purpose, we are recording all the topics by putting -a and then excluding using yaml file.